trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/PreprocessorHandler.java

```
@@ -144,12 +141,6 @@ public Result visit(final IDispatcher main, final IASTPreprocessorMacroDefinitio
 
 	@Override
 	public Result visit(final IDispatcher main, final IASTPreprocessorPragmaStatement node) {
-		if (mIgnorePreprocessorPragmas) {
-			return new SkipResult();
-		}
-		final String msg = "PreprocessorHandler: Not yet implemented: " + node.toString();
-		final ILocation loc = mLocationFactory.createCLocation(node);
-		mReporter.unsupportedSyntax(loc, msg);
 		return new SkipResult();

```

same here: we should not silently ignore preprocessor pragmas